frontend/src/modules/history/components/Translation.js

```diff
@@ -123,6 +123,24 @@ export class TranslationBase extends React.Component<InternalProps, State> {
         return 'Not reviewed yet';
     }
 
+    renderUserAvatar() {
+        const { translation } = this.props;
+
+        if (!translation.uid) {
+            return <img src={ translation.user_gravatar_url_small } alt="User avatar" />;
```

Comment:
If translation lacks `uid`, it also lacks `user_gravatar_url_smal`. Which results it:
<img width="555" alt="Screenshot 2019-10-16 at 14 14 53" src="https://user-images.githubusercontent.com/626716/66918312-57d8f700-f01f-11e9-850b-34125e6dcd48.png">

Obviously, that's not great. To mitigate the problem, we can either:
1. not show the avatar at all (not even the `<div.avatar>`), which will make those translations stand out more
1. use `/static/img/anon.jpg` - our fallback when user doesn't have an image set
1. find a better image to be used for strings without Translation authors (i.e. imported from version control)

Which option would you prefer? I'd probably go with #2 for now.

Reply:
Yes, I also prefer #2